sdk-springboot/sdk-spring-core/src/test/java/io/dapr/spring/data/AbstractBaseIT.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2024 The Dapr Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package io.dapr.spring.data;
+
+import org.testcontainers.containers.Network;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+@Testcontainers
+public abstract class AbstractBaseIT {
```

Comment:
@artur-ciocanu we will need to move this to `sdk-tests` there is an IT package. Look at how I moved the test in the testcontainers-dapr module -> https://github.com/dapr/java-sdk/pull/1085/files#diff-3567b77e1a2a23e9200900907bd789529d99ccf8968b54fd0f8fc8eac3c63d24

Reply:
Moved